src/server/scripts/Spells/spell_generic.cpp

```diff
@@ -1837,6 +2006,22 @@ class spell_gen_obsidian_armor : public AuraScript
     }
 };
 
+// 55342 Mirror Image
+class spell_gen_one_tick_dummy : public AuraScript
```

Comment:
55342 doesn't have any effect with SPELL_AURA_PERIODIC_DUMMY.
EFFECT_1 is SPELL_EFFECT_DUMMY, so probably changed and can now be removed.